app/design/adminhtml/default/default/template/catalog/product/edit/price/tier.phtml

```diff
@@ -56,13 +56,13 @@
                 <th class="last"><?php echo Mage::helper('catalog')->__('Action') ?></th>
             </tr>
         </thead>
-        <tbody id="<?php echo $_htmlId ?>_container"></tbody>
         <tfoot>
             <tr>
                 <td <?php if (!$_showWebsite): ?>style="display:none"<?php endif; ?>></td>
                 <td colspan="4" class="a-right"><?php echo $this->getAddButtonHtml() ?></td>
             </tr>
         </tfoot>
+        <tbody id="<?php echo $_htmlId ?>_container"></tbody>
```

Comment:
Why is `tbody` below `tfoot`?

Reply:
Quite a few files in the core have `tbody` below `tfoot`. `app\design\adminhtml\default\default\template\customer\tab\view\sales.phtml` is one. [Browser](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/tfoot) renders just fine.

Reply:
I moved it because the [HTML Validator](http://users.skynet.be/mgueury/mozilla) Firefox plugin ask this change.